### ISSUE TITLE
'tensorflow.python.keras.api._v1.keras.losses' has no attribute 'Reduction'

### ISSUE THREAD

**Issue by subhankar-ghosh** [tensorflow/tensorflow#33356]
I am using Huber loss implementation in tf.keras in tensorflow 1.14.0 as follows:

```
huber_keras_loss = tf.keras.losses.Huber(
        delta=delta,
        reduction=tf.keras.losses.Reduction.SUM,
        name='huber_loss'
    )
```

I am getting the error AttributeError: module 'tensorflow.python.keras.api._v1.keras.losses' has no attribute 'Reduction'

I have tried using tf.losses.Reduction, tf.compat.v2.losses.Reduction nothing seems to work.

Did tensorflow remove Reduction from tf.keras.losses, it is strange if they did so because their documentation still shows: https://www.tensorflow.org/versions/r1.14/api_docs/python/tf/keras/losses/Huber#args

**Comment by MarkDaoust** [comment]
Hi @subhankar-ghosh, 

Can you please retry this on the `tf-nightly` release, and post the full code to reproduce the problem?

The 1.14 release was cut at the beginning of June, and it's entirely possible that this is fixed.

**Comment by AlexKai1** [comment]
can’t solve this problem.

**Comment by yashk2810** [comment]
This seems fixed in TF 2.1. Please reopen if you are still hitting errors.
